src/cascadia/LocalTests_TerminalApp/TabTests.cpp

```diff
@@ -276,12 +275,8 @@ namespace TerminalAppLocalTests
             ]
         })" };
 
-        VerifyParseSucceeded(settingsJson0);
-        auto settings0 = winrt::make_self<implementation::CascadiaSettings>(false);
+        CascadiaSettings settings0{ winrt::hstring{ til::u8u16(settingsJson0) } };

```

Comment:
weird: this one grew an `hstring` conversion but the other two didn't need it ...

Reply:
(the ones lower in the file)